release-notes/opendistro-for-elasticsearch-job-scheduler.release-notes-1.10.1.1.md

```diff
@@ -1,6 +1,11 @@
-## 2020-09-08 Version 1.10.1.0
+## 2020-10-08 Version 1.10.1.1
```

Comment:
You shouldn't modify an existing release note, these are immutable once a release goes out. You need to keep both the 1.10.1.0 and add a new 1.10.1.1

Reply:
Make sense, and thanks for the explanation. I have restored the existing one.